cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3624,8 +3709,8 @@ cron_schedule = 0 1 * * *
 description = Generates the cs_authentication_usual_location.csv file with the Usual Login Location details for users. \
 \
 Data Collection - VPN data mapped with authentication data-model.
-dispatch.earliest_time = -7d@h
-dispatch.latest_time = now
+dispatch.earliest_time = -16d@d
```

Comment:
Shall we make ti for 1 month, I'm not sure?

Reply:
We have to somehow rank it as well (We can do inside eventstats command itself)
* and restrict by number of allowed countries to login as well. (lets say default to 3 max)
* this will be additional condition over percerage inside alerts.